packages/reports/tests/integration/components/filter-values/dimension-select-test.ts

```diff
@@ -363,8 +364,42 @@ module('Integration | Component | filter values/dimension select', function (hoo
         el.querySelector('.filter-values--dimension-select__option-value')?.textContent?.trim(),
         el.querySelector('.filter-values--dimension-select__option-context')?.textContent?.trim(),
       ]),
-      [['65 and over', 'id: 10']],
+      [
+        ['"over"', ''],
+        ['65 and over', 'id: 10'],
+      ],
       '`dimension-select` fetches values for new dimension field'
     );
   });
+
+  test('testing manual filter', async function (this: TestContext, assert) {
+    assert.expect(2);
+    this.filter = this.fragmentFactory.createFilter('dimension', 'bardOne', 'age', { field: 'id' }, 'in', []);
+    this.onUpdateFilter = (changeSet: Partial<FilterFragment>) => {
+      this.set('filter.values', changeSet.values);
+    };
+
+    await render(TEMPLATE);
+
+    await selectSearch('.filter-values--dimension-select__trigger', 'abc');
+    assert.deepEqual(
+      findAll('.filter-values--dimension-select__option').map((el) => [
+        el.querySelector('.filter-values--dimension-select__option-value')?.textContent?.trim(),
+        el.querySelector('.filter-values--dimension-select__option-context')?.textContent?.trim(),
+      ]),
+      [['"abc"', '']]
+    );
+
+    this.onUpdateFilter = (changeSet: Partial<FilterFragment>) => {
+      assert.deepEqual(
+        changeSet.values,
+        [...this.filter.values, 'xyz'],
+        'changeSet should update after clicking on a value from dropdown'
+      );
+    };
+
+    await clickTrigger();
+    await typeIn('.filter-values--dimension-select__trigger input', 'xyz');
```

Comment:
Use `selectSearch` or if that doesn't work then `fillIn` because [`typeIn`](https://github.com/emberjs/ember-test-helpers/blob/master/API.md#typein) mimics slowly typing characters